schemas/2.2/schema.json

```
@@ -21,6 +21,18 @@
         "2.2"
       ]
     },
+    "instance":{
+      "description": "Metadata about the instance",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "description": "The utf8 encoded name",
```

JSON should always be UTF-8 encoded AFAIK, so that's a bit redundant. I'd go for being a bit more verbose too:

```suggestion
          "description": "If supported by the software the administrator configured name of this instance",
```